docs/guides/custom-redirects.mdx

```diff
@@ -199,11 +211,11 @@ export default function Home() {
     <div>
       <h1>Welcome!</h1>
 
-      <SignInButton signInFallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
+      <SignInButton fallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
```

Comment:
wait, why is this being changed in all of these button components? the doc doesn't ever define a `fallbackRedirectUrl` ? only `signUpFallbackRedirectUrl` and `signInFallbackRedirectUrl`

Reply:
I'll add those to the doc. You can see [this discussion](https://clerkinc.slack.com/archives/C01QFRQNHSN/p1713903676149769) and [this one](https://clerkinc.slack.com/archives/C04KENP2GFL/p1713901849337169) for info on why we use `fallback` instead of `signInFallback`, etc, on some components